.circleci/config.yml

```diff
@@ -1,318 +1,113 @@
 version: 2.1
 
 orbs:
-  build-tools: circleci/build-tools@2.6.2
-  jq: circleci/jq@volatile
-  node: circleci/node@dev:alpha
-  orb-tools: circleci/orb-tools@8.27.3
+  node: circleci/node@<<pipeline.parameters.dev-orb-version>>
+  orb-tools: circleci/orb-tools@9.0
+
+# Pipeline parameters
+parameters:
+  # These pipeline parameters are required by the "trigger-integration-tests-workflow"
+  # job, by default.
+  run-integration-tests:
+    type: boolean
+    default: false
+  dev-orb-version:
+    type: string
+    default: "dev:alpha"
+
+integration-tests: &integration-tests
+  [
+    integration-test-install-linux,
+    integration-test-install-mac,
+    node-test-job
+  ]
 
 jobs:
-  test:
-    parameters:
-      executor:
-        type: executor
-        default: node/default
-
-      node-version:
-        type: string
-        default: latest
-
-      node-alpine-version:
-        type: string
-        default: nodejs-current
-
-      npm-version:
-        type: string
-        default: latest
-
-      yarn-version:
-        type: string
-        default: ""
-
-    executor: <<parameters.executor>>
+# Install NodeJS into a non-node container.
+  integration-test-install-linux:
+    docker:
+      - image: cimg/base:stable
     steps:
       - checkout
-      - build-tools/install-ci-tools
-
       - node/install:
-          node-version: <<parameters.node-version>>
-          node-alpine-version: <<parameters.node-alpine-version>>
-          npm-version: <<parameters.npm-version>>
-          yarn-version: <<parameters.yarn-version>>
-          debug: true
-
-# yaml anchor filters
-integration-dev_filters: &integration-dev_filters
-  branches:
-    ignore: /.*/
-  tags:
-    only: /integration-.*/
-
-integration-master_filters: &integration-master_filters
-  branches:
-    ignore: /.*/
-  tags:
-    only: /master-.*/
+          node-version: "13.11.0"
```

Comment:
I think it'd be good if we could test installation with `node-version: latest`? I gave it a quick test but the installation failed for me:

```
Downloading: https://nodejs.org/download/release/vlatest/node-vlatest-linux-x64.tar.gz
curl: (22) The requested URL returned error: 404 
```

Reply:
Alright, just got some updates pushed in.

"latest" was the default if you **did not** supply a version but it was not setup to accept "latest" as a version. In NodeJS land, "latest" is actually called "current". So, you may now specify nothing, "latest", or "current" and all of those will get you the latest version of NodeJS.

EXCEPT for the test job. The executor can not perform any logic to fetch the version at run time, so the version selected for the docker executor must be a full version string.